src/parser/parse_tree.h

```diff
@@ -2252,6 +2252,7 @@ struct pt_expr_info
 
 #define PT_EXPR_INFO_GROUPBYNUM_LIMIT 32768	/* flag that marks if the expression resulted from a GROUP BY ... LIMIT
 						 * statement */
+#define PT_EXPR_INFO_DONT_AUTOPARAM 65536	/* don't auto parameterize expr at qo_do_auto_parameterize() */
```

Comment:
I think the macro name PT_EXPR_INFO_DO_NOT_AUTOPARAM is better than PT_EXPR_INFO_DONT_AUTOPARAM because as the common coding convention DONT is seldom used than DO_NOT.

Reply:
I agree

Reply:
I fixed it